mpd/base.py

```diff
@@ -548,6 +551,77 @@ def _read_lines(self):
             yield line
             line = self._read_line()
 
+    def _read_numbytes(self, numToRead):
+        retVal = bytearray()
+        while numToRead > 0:
+            readResult = self._rbfile.read(numToRead)
+            if len(readResult) == 0:
+                break
+            retVal.extend(readResult)
+            numToRead -= len(readResult)
+        return bytes(retVal)
```

Comment:
python naming style:

```suggestion
    def _read_bytes(self, amount):
        data = bytearray()
        while amount > 0:
            result = self._rbfile.read(amount)
            if len(result) == 0:
                break
            data.extend(result)
            amount -= len(result)
        return bytes(data)
```